doc/unison-manual.tex

```diff
@@ -1535,6 +1535,23 @@
 case, all actions that have been selected for a set of changes
 currently being displayed will be discarded.
 
+When unison reads roots from a profile, spaces and tabs immediately
```

Comment:
I'm fine overall, just one last nitpick. This sentence is about roots specifically, but it's actually true for all pathnames. Also, there is a more generic sentence about the same thing ~40 lines earlier (line 1501).

Reply:
Good spot on earlier text.   I merged the space caution into that and simplified a lot.  Needs a strings regen, after an issue on master is resolved (either needs a strings regen or we have some odd environment regen issue again).

Reply:
strings issues are resolved.   I'll merge this tomorrow or sooner if I hear you are happy with it.